Cannot compile android export templates with Mono in the master branch

**Godot version:**
Master branch 3b4fad4
Also tried 3.2 stable before (more in description)

**OS/device including version:**
Windows 10 64-bit
VS 2019 Native tools Command Prompt x64

**Issue description:**
Firstly I compiled the stable branch of 3.2 with mono enabled (following the instructions)
Also compiled mono for android (https://github.com/godotengine/godot-mono-builds)
Then in VS 2019 Tools I'm compiling templates: set env variables, running command scons p=android target=release module_mono_enabled=yes android_arch=armv7 mono_prefix="Path-To-Compiled-Monos\android-armeabi-v7a-release"
And on file `Godot-Source-Path\modules\mono\mono_gd\android_mono_config.gen.cpp` I have this issue:
```
[ 18%] Compiling ==> modules\mono\mono_gd\android_mono_config.gen.cpp
=====
"modules\\mono\\mono_gd\\android_mono_config.gen.cpp:21:27: error: member reference base type 'uint8_t *' (aka 'unsigned char *') is not a structure or union
Compression::decompress(w.ptr(), config_uncompressed_size, config_compressed_data,
~^~~~
modules\\mono\\mono_gd\\android_mono_config.gen.cpp:24:34: error: member reference base type 'uint8_t *' (aka 'unsigned char *') is not a structure or union
if (s.parse_utf8((const char *)w.ptr(), data.size())) {
~^~~~
2 errors generated.
"
=====
scons: *** [modules\mono\mono_gd\android_mono_config.gen.android.opt.armv7.neon.o] Error 1
scons: building terminated because of errors.
```
I've started to search in source code and found the code, what generates the `android_mono_config.gen.cpp`.
It was generated by `Godot-Source-Path\modules\mono\build_scripts\make_android_mono_config.py`

Before I tried to compile stable 3.2 Godot branch. But there I had another problems.
Is was about 20 errors generated on compiling file `Godot-Source-Path\modules\mono\csharp_script.cpp`, but in master this file compiles good. And `android_mono_config.gen.cpp` generation in 3.2 was different with current master too.
So I hope its not so big problem and you will fix it fast or maybe help me if its problem in my compilation process..
I hope I wrote enough info... Thanks in advance :)

**Steps to reproduce:**
1. Compile 3b4fad4 Godot master
2. Compile Mono for Android from https://github.com/mono/mono (master branch)
3. Compile Android Templates with scons p=android target=release module_mono_enabled=yes android_arch=armv7 mono_prefix="Path-To-Compiled-Monos\android-armeabi-v7a-release"

--- a/modules/mono/build_scripts/make_android_mono_config.py
+++ b/modules/mono/build_scripts/make_android_mono_config.py
@@ -38,10 +38,10 @@ String get_godot_android_mono_config() {
 	Vector<uint8_t> data;
 	data.resize(config_uncompressed_size);
 	uint8_t* w = data.ptrw();
-	Compression::decompress(w.ptr(), config_uncompressed_size, config_compressed_data,
+	Compression::decompress(w, config_uncompressed_size, config_compressed_data,
 			config_compressed_size, Compression::MODE_DEFLATE);
 	String s;
-	if (s.parse_utf8((const char *)w.ptr(), data.size())) {
+	if (s.parse_utf8((const char *)w, data.size())) {
 		ERR_FAIL_V(String());
 	}
 	return s;

